django/forms/forms.py

```diff
@@ -578,7 +597,22 @@ def as_widget(self, widget=None, attrs=None, only_initial=False):
             name = self.html_name
         else:
             name = self.html_initial_name
-        return force_text(widget.render(name, self.value(), attrs=attrs))
+
+        args = getargspec(widget.render)[0]
```

Comment:
This will need updating to account for #4846 (`getargspec` is deprecated in Python 3 and will soon be removed).